Stop executing YAML tags for mkdocs_theme.yml, warn about third-party projects (#3465)
mkdocs/mkdocs: docs/user-guide/choosing-your-theme.md

A list of third party themes can be found at the [community wiki] page and [the ranked catalog][catalog]. If you have created your own, please add them there.

> WARNING: Installing an MkDocs theme means installing a Python package and executing any code that the author has put in there. So, exercise the usual caution; there's no attempt at sandboxing.

Reviewer: ```suggestion
WARNING: By installing an MkDocs plugin you are installing a Python package and executing any code embedded by the author. There's no attempt at sandboxing, so exercise the usual caution.
```
